src/router/index.js

```diff
@@ -187,7 +187,7 @@ const router = createRouter({
     } else if (savedPosition) {
       return savedPosition
     } else if (to.meta.scrollPosition) {
-      return to.meta.scrollPosition
+      return window.scrollTo(to.meta.scrollPosition.x, to.meta.scrollPosition.y)
```

Comment:
Need to figure out why `vue-router` doesn't do a scroll internally based of `meta.scrollPosition`